lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs

```diff
@@ -145,9 +147,17 @@ internal sealed class REEFIMRUClient : IIMRUClient
                 .SetJobIdentifier(jobDefinition.JobName)
                 .Build();
 
-            _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
+            _httpEndPoint = _reefClient.SubmitAndGetDriverUrl(imruJobSubmission);
 
             return null;
         }
+
+        /// <summary>
+        /// DriverHttpEndPoint returned by IReefClient after job submission
+        /// </summary>
+        public IDriverHttpEndpoint DriverHttpEndpoint
```

Comment:
This will create lots of code changes inother projects already using the
API that is already unstable and will change in future.

On Mon, Oct 19, 2015 at 11:17 AM, Julia Wang <notifications@github.com>
wrote:

> In lang/cs/Org.Apache.REEF.IMRU/OnREEF/Client/REEFIMRUClient.cs
> <https://github.com/apache/incubator-reef/pull/573#discussion_r42406559>:
>
> >
> >              return null;
> >          }
> > +
> > +        /// <summary>
> > +        /// DriverHttpEndPoint returned by IReefClient after job submission
> > +        /// </summary>
> > +        public IDriverHttpEndpoint DriverHttpEndpoint
>
> Then why not return it from REEFIMRUClient Submit method? It currently
> returns null and no one is using the return value.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/573/files#r42406559>.
>